packages/oui-checkbox/_mixins.less

```diff
@@ -221,7 +221,7 @@
     }
 
     .@{checkbox-selector}__input:not(:disabled) + .@{checkbox-selector}__label-container:hover,
-    .@{checkbox-selector}__input:checked:not(:disabled) + .@{checkbox-selector}__label-container:hover {
+      .@{checkbox-selector}__input:checked:not(:disabled) + .@{checkbox-selector}__label-container:hover {
```

Comment:
Is this really something stylelint expect ? Not sure to like this additionnal indentation, I know it's purely personal...